sdk/azcore/policy_telemetry.go

```diff
@@ -64,6 +64,8 @@ func NewTelemetryPolicy(o *TelemetryOptions) Policy {
 		b.WriteString(o.Value)
 		b.WriteRune(' ')
 	}
+	b.WriteString(UserAgent)
```

Comment:
How does the user opt out of adding this?

Reply:
See line 49 where telemetry is disabled.  It's all-or-nothing, they  can't opt out of parts of it.

Reply:
Ah got it. Thanks!